File: 1-two-sum/1-two-sum.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
        var d = new Dictionary<int, int>();
        for (int i = 0; i < nums.Length; i++) {
            if (d.ContainsKey(nums[i])) {
                return new int[2] {d[nums[i]], i};
            }
            d.TryAdd(target - nums[i], i);
        }
        return new int[2] {-1, -1};
    }
}

File: 1-two-sum/1-two-sum.py
class Solution:
    def twoSum(self, nums:List[int],target: int) -> List[int]:
        list = []
        for i, x in enumerate(nums):
            for j, y in enumerate(nums):
                if i != j:
                    if x+y == target:
                        list.append(i)
                        list.append(j)
                        return list
        
                        
        